custom-namespaces.yaml

```
@@ -1053,3 +1053,13 @@ relay:
       type: table_view
       tables:
         - table: mozdata.relay.active_subscription_ids
+fivetran_costs:
```

```suggestion
fivetran:
```
should we just call it `fivetran`? We might want to add more views/explores that are related to Fivetran but not related to cost specifically in the future?

---

good point ... but the dataset is now named fivetran_costs ... should that be renamed as well or could I put multiple datasets into this namespace?